.NET/Microsoft.Recognizers.Text/Matcher/AcAutomaton.cs

```diff
@@ -5,16 +5,15 @@ namespace Microsoft.Recognizers.Text.Matcher
 {
     public class AcAutomaton<T> : AbstractMatcher<T>
     {
-        protected readonly AaNode<T> root = new AaNode<T>();
```

Comment:
Extract the changes related to the field rename/promote to property into another PR, AND keep the other changes (reordering, spacing, new lines, etc.) in this PR.

Reply:
Extracted changes to new branch

Reply:
By "extract" I mean: Revert all changes related to the field rename/promote to property in this branch. and move them into another branch.

Reply:
Forgot to push, now its ok!